core/src/main/java/com/linecorp/armeria/server/cors/ChainedCorsPolicyBuilder.java

```diff
@@ -36,6 +38,7 @@ public final class ChainedCorsPolicyBuilder extends AbstractCorsPolicyBuilder {
     private final CorsServiceBuilder serviceBuilder;
 
     ChainedCorsPolicyBuilder(CorsServiceBuilder builder) {
+        super(Collections.singletonList("*"));
```

Comment:
Should we cache this value in a `static` field?

Reply:
Sure 👍